### ISSUE TITLE
Banner ecosystem

### ISSUE THREAD

**Issue by lhorakov** [OpenSCAP/website#158]
From banner with different graphical tools we are going to remove OpenSCAP Base and Daemon which do not have a role in this info-graphics.

**Comment by mpreisler** [comment]
SCAP Workbench and OAA both use OpenSCAP Base. Perhaps we can connect it instead of removing it.

**Comment by lhorakov** [comment]
Yeah, we could. I'm looking on banner but don't know how I will do it. There is no more space for it...

**Comment by mpreisler** [comment]
What is the reasoning to change the banner? Is this an outcome of UX testing?

**Comment by lhorakov** [comment]
Yes it is. When they start to describe it (I'm not asking them to do so, they are doing it themselves), they are just trying to include in the process Base and Daemon, and I put there those 2 only because they are also in tools, but they aren't part of info-graphics actually. They do not have there some graphical tubes...

**Comment by mpreisler** [comment]
Both of them are sort of in the background of things, used by other tools. I think we can keep them in the background.

**Comment by isimluk** [comment]
I feel it is good thing to have base and daemon on the graphics. I guess it is a right thing for people to grasp that they understand how graphical tools fit together while concluding they do not understand another bit, they need to learn about.

tldr, removing the base and daemon does not fix the problem, it just makes it less visible.

Just my €0.02. I haven't been there.

**Comment by lhorakov** [comment]
Thank you for your cents :D.
It would fix the problem because people wouldn't imagine what those tools are doing but as you both argued, I will let it as it is. Now I'm just thinking if there is something very easy to add there to make it clearer or not...

**Comment by lhorakov** [comment]
I'm letting it be as it is.
